content/responses/file--mini.yml

```diff
@@ -28,3 +28,51 @@ allOf:
         x-box-field-variant: 1
         description: The name of the file
         example: "Contract.pdf"
+
+      metadata:
```

Comment:
Unless metadata is _always_ returned for every mini representation, then we shouldnt have this here

Reply:
got it. Will move